jcl/src/java.base/share/classes/java/lang/invoke/MethodHandleResolver.java

```diff
@@ -23,6 +23,7 @@
 package java.lang.invoke;
 
 import java.lang.invoke.MethodHandles.Lookup;
+import static java.lang.invoke.MethodHandles.Lookup.IMPL_LOOKUP;
 import java.util.ArrayList;
 import java.util.Objects;
```

Comment:
It's preferred to separate the static and non-static imports. In most existing files, the preferred format is to declare static imports first.

```suggestion
package java.lang.invoke;

import static java.lang.invoke.MethodHandles.Lookup.IMPL_LOOKUP;

import java.lang.invoke.MethodHandles.Lookup;
import java.util.ArrayList;
import java.util.Objects;
```

Reply:
Addressed comment.